app/models/manager.rb

```diff
@@ -0,0 +1,3 @@
+class Manager < ActiveRecord::Base
+	has_one :team
```

Comment:
Close! I would say that a manager `belongs_to` a team, and a team `has_many` managers. This stuff is a little bit tricky